app/views/rooms/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<%@rooms.each do |room|%>
+<h2>Suite Name: <%=room.suite%></h2>
+<p>Nightly Rate: $<%=room.rate%></p>
+<p>Hotel: <%=room.hotel.name%></p>
+<p>Number of guests who have stayed: <%=room.guests.count%></p>
+<%end%>
```

Comment:
If we think about line `5` where you are doing `room.guests.count` and think about what it's doing. On that line, it's taking `room.guests` and then calling the ActiveRecord aggregate method `count` on it. This is performing a database call, and because this is hitting the database, this code needs to be in the `Room` model.

I just want to compare that to line `4`, where you are doing `room.hotel.name`. On this line, you aren't calling the database because when we called on all the `@rooms` in the controller, those rooms, came with the hotel data that is linked to each room, and each of those hotels came with its attributes, including the `name`. No additional database call is being made there.